README.md

```diff
@@ -1,44 +1,88 @@
 # Formal Ledger Specifications
 
-This repository contains the formal ledger specifications that are intended to eventually replace the existing formal specifications of the Cardano ledger found [here](https://github.com/IntersectMBO/cardano-ledger). This project is currently incomplete and work in progress.
+This repository contains the formal ledger specifications for the Cardano blockchain,
+written in [Agda](https://wiki.portal.chalmers.se/agda/pmwiki.php). These
+specifications are executable, and Haskell code can be extracted from the Agda source
+code and run for conformance testing.
 
-This repository currently contains the work-in-progress specification for Cardano (up to and including the Conway era). The specification is executable and contains some documentation in the form of a PDF document. It can be built by following the steps in [Building](#building).
+The project consists entirely of literate Agda code from which we generate documentation,
+viewable in a web browser, that replaces the TeX/PDF-based specifications of previous eras.
```

Comment:
I think TeX is a technical detail irrelevant at this point.
```suggestion
viewable in a web browser, that replaces the PDF-based specifications of previous eras.
```

Reply:
Actually I would split the phrase in two:

```
The project consists entirely of literate Agda code from which we generate web-based documentation.
This replaces the PDF-based documentation of previous eras.
```
And maybe add a footnote with a link to the README in cardano-ledger where the pdfs can be found.